recipes/runit_systemd.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright:: Copyright (c) 2015 Opscode, Inc.
+#
+# All Rights Reserved
+#
+
+project_name = node['enterprise']['name'].clone
+project_name.gsub!(/_/, '-') if project_name == 'private_chef'
+unit_name = "#{project_name}-runsvdir-start.service"
```

Comment:
Yeah so then `project_name = 'private-chef' if project_name == 'private_chef'` seems much clearer, and should really have a comment on it, because its pretty wtf (and the original should do that as well).